src/net/sourceforge/kolmafia/svn/SVNManager.java

```diff
@@ -515,8 +515,7 @@ private static boolean validateRepo(SVNURL repo, boolean quiet) {
 
```

Comment:
hm, it looks like failed is potentially being cleared depending on loop order, right before this (in the dependencies.txt check; should probably read failed |= ...).

(I was curious whether we could provide a better error message for _why_ it failed validation, in addition to what as you have here.)

Reply:
I couldn't really get a useful debugger run without actually changing a repository and that was more work than just making the change.   I wondered about a better message but I wasn't sure I wanted to understand the SVN code well enough to actually make an intentional change.  Worth revisiting if it is not made obsolete by adding git support to KoLmafia or refactoring how scripts get installed and updated.